### ISSUE TITLE
[FR] Power Cable Bus

### ISSUE THREAD

**Issue by OmegaHaxors** [Electrical-Age/ElectricalAge#914]
![2019-04-12_22 34 42](https://user-images.githubusercontent.com/16057503/56075220-deacce80-5d73-11e9-809c-f6a77cd9b6f7.png)

On the left you have an existing system, each junction has a switch. It's pretty clunky, but mechanically it functions.

On the right there's a cable bus. From one centralized location, you can adjust which machine gets power either by using the GUI or by using a signal bus. Current and voltage limits are consistent to a single cable, so this won't allow you to deliver more power than normal, but it will give a centralized location to manage all of the power on your cable.

**Comment by jrddunbr** [comment]
First, those colors hurt my eyes :P

Sounds like an interesting concept, but I'm curious how effective that will be, particularly how it will be implemented in the internals of the mod.

**Comment by OmegaHaxors** [comment]
Internals would be easy: The engine would ignore any 'deactivated' colors as if they were Giga Resistors, and the control panel will determine which colors are allowed, via a signal bus input.

![2019-04-12_23 05 19](https://user-images.githubusercontent.com/16057503/56075422-88da2580-5d77-11e9-95a4-a7abaf9a8455.png)

Here's an example usage: Easy Boost Converter. The blue and black path activate until the inductor charges, then the green path activates to discharge. If for whatever reason power needs to be sustained, both the blue and green path can be disabled and the red path starts instead.

Not only is such a system easy and compact, but it also encourages smart signal use.

**Comment by jrddunbr** [comment]
OK, and what about when there are multiple SigProcs/light panels? Do they share a combined signal voltage over the length of the cable in a special signal channel?

**Comment by OmegaHaxors** [comment]
Could be done similar to how lights are. Three modes: Bigger Smaller and Toggle.

**Comment by jrddunbr** [comment]
So you mean to say that this cable is effectively a power cable combined with a SigBus cable? That would be neat.

**Comment by OmegaHaxors** [comment]
Kind of? It's more like cables paths you can control via a signal.
For example, to activate red and blue paths, you input a red and blue signal of True.

**Comment by jrddunbr** [comment]
Yeah, but you have a signal for each cable color. So it's 16 signals.

**Comment by OmegaHaxors** [comment]
Signal-controlled power cables.

**Comment by jrddunbr** [comment]
![IMG_20190413_022600](https://user-images.githubusercontent.com/14130505/56075672-4d018900-5d94-11e9-86a4-f72dd2e9692a.jpg)
